addons/payment/wizards/payment_acquirer_onboarding_templates.xml

```diff
@@ -42,12 +42,10 @@
                                     </a>
                                 </p>
                             </div>
-                            <div attrs="{'invisible': [('payment_method', '!=', 'stripe')]}">
+                            <div invisible="1">
```

Comment:
In stable, do we have the right to do it ?

If one extends this view and uses those fields in an xpath expr, it may cause a failure. You may ask why one would do that ? 
If you think it's legit to remove the whole div, then I'll do it.